onnx/reference/ops/op_lp_pool.py

```diff
@@ -0,0 +1,360 @@
+# SPDX-License-Identifier: Apache-2.0
+# pylint: disable=C0200,R0912,R0913,R0914,R0915,R0916,R1702,W0221
+
+import numpy as np
+
+from ._op_common_pool import CommonPool
+
+
+class LpPool(CommonPool):
+    def _run(  # type: ignore
+        self,
+        x,
+        auto_pad=None,
+        ceil_mode=None,
+        dilations=None,
+        kernel_shape=None,
+        p=None,
+        pads=None,
+        strides=None,
+    ):
+        if (
+            dilations is not None
+            and (min(dilations) != max(dilations) or min(dilations) != 1)
+        ) or (
+            strides is not None and (min(strides) != max(strides) or min(strides) != 1)
+        ):
+            return self._lp_pool(
+                x,
+                auto_pad=auto_pad,
+                ceil_mode=ceil_mode,
+                dilations=dilations,
+                kernel_shape=kernel_shape,
+                p=p,
+                pads=pads,
+                strides=strides,
+            )
+
+        return CommonPool._run(
```

Comment:
So, does this not handle dilations/stride? Trying to understand the special case above.